### PR TITLE
Skip app check for ringtools

### PR DESCRIPTION
**Pull request by AaronDewes**
Even with the CORS fix, the app was still shown as "starting". So this will work as a temporary workaround for 0.0.5

### CODE REVIEW

**Comment by pwltr** [comment]
The same logic should be added to `AppStoreApp.vue` that also contains the polling. A fallback idea to work around this CORS issue would be to work with the manager to check for container status with something like `docker container inspect -f '{{.State.Status}}' ringtools-main-1`. It's not as good an end-to-end check as the current HTTP request polling but it would work around the CORS/CORP issues. It also might come in handy for apps that don't have a web UI.

**Comment by pwltr** [comment]
We should also add some screenshots for this app.

**Comment by AaronDewes** [comment]
Sure, they can be added later. Will update the AppStoreApp code though.

File: src/components/InstalledApp.vue [hunk 1]
```diff
@@ -162,7 +164,7 @@ export default defineComponent({
     },
     async pollOfflineApp() {
       this.checkIfAppIsOffline = true;
-      if (this.id === 'bluewallet') this.checkIfAppIsOffline = false;
+      if (skipCheckApps.includes(this.id)) this.checkIfAppIsOffline = false;
```
Comment (pwltr) [inline src/components/InstalledApp.vue:167]: Add the same logic to `AppStoreApp.vue`

**Review by pwltr — CHANGES_REQUESTED** [review]

**Review by pwltr — APPROVED** [review]

**Review by WilliamConnatser — APPROVED** [review]

**Pull request merged**